tools/osbuild-image-test

```diff
@@ -400,7 +400,7 @@ def run_tests_cases(args):
     success = True
     if not tests:
         print(f"{RED}FAIL{RESET}: no tests to run")
-        return False
+        return 0
```

Comment:
This is also the return code for `main()` which is also the exit code for the script, so it should be non-zero!

Reply:
:facepalm: Omg, yes, I wanted to put `exit 1` actually, thanks!